modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -799,8 +860,11 @@ private void _setSchemaDescription(
 	private final ObjectEntryOpenAPIResource _objectEntryOpenAPIResource;
 	private final ObjectEntryOpenAPIResourceProvider
 		_objectEntryOpenAPIResourceProvider;
+	private final ObjectFieldLocalService _objectFieldLocalService;
 	private final ObjectRelationshipLocalService
 		_objectRelationshipLocalService;
 	private final OpenAPIResource _openAPIResource;
+	private final Set<String> _readOnlyFieldNames = SetUtil.fromArray(
+		"dateCreated", "dateModified", "id");
```

Comment:
I think "id" exists as a field and it could be removed from this Set as it could be processed as the rest of the internal f ields